.github/workflows/osl_sync_main.yml

```diff
@@ -21,12 +21,25 @@ jobs:
       pull-requests: write
 
     steps:
+      - name: "Setup pnpm"
+        uses: pnpm/action-setup@v3
+        with:
+          version: 9.3.0
+
+      - name: "Setup Node"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.13.1
```

Comment:
is this particular version of Node selected due any reason?
Internally we use 22.18.0

Reply:
ok